elide-model-config/src/main/java/com/yahoo/elide/modelconfig/jsonformats/ElideNameFormatAttr.java

```diff
@@ -19,9 +19,11 @@
  * </p>
  */
 public class ElideNameFormatAttr extends AbstractFormatAttribute {
-    private static final String NAME_FORMAT_REGEX = "^[A-Za-z][0-9A-Za-z_]*$";
+    public static final String NAME_FORMAT_REGEX = "^[A-Za-z][0-9A-Za-z_]*$";
```

Comment:
Consider pre-formatting the regex.
```suggestion
    public static final Pattern NAME_FORMAT_REGEX = Pattern.compile("^[A-Za-z][0-9A-Za-z_]*$");
```